### ISSUE TITLE
TODOs tutorial is broken

### ISSUE THREAD

**Issue by troygoode** [weepy/o_O#8]
http://weepy.github.com/o_O/examples/todos/index.html

Error:

`Uncaught TypeError: Object function o_O() {} has no method 'collection'`

Being thrown from:

``` javascript
var ViewModel = function(todos) {
        var self = this;
        //map array of passed in todos to an observableArray of Todo objects
        self.todos = o_O.collection(todos)
```

Sorry, didn't have time to investigate further yet.

**Comment by weepy** [comment]
thanks - the website needs upgrading to the latest version of the
example and the library.

**Comment by troygoode** [comment]
Is fixed by pull request in issue #10.
